### PR TITLE
[risk=low] RW-12040 Allow workspace deletion

### PR DESCRIPTION
**Pull request by Qi77Qi**
Remove the warning that prevents users from deleting their workspaces when there's app in the workspace.

New banner looks like this when trying to delete workspace
Screenshot 2024-03-11 at 4 42 54 PM

Note if you have a workspace that previously has orphaned Sam resources for apps. Workspace deletion will still fail, and eng will have to step in to help users to clean up

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

**Review by yonghaoy — CHANGES_REQUESTED** [review]
Two more file you need to modify: see https://github.com/all-of-us/workbench/pull/8396/files

**Review by yonghaoy — APPROVED** [review]
LG. All a risk level in RR description for that release ticket can grab risk level. 
e.g. [risk=low]

**Comment by Qi77Qi** [comment]
@jmthibault79 going to merge this...lmk if u have any comments and we can address retrospectively

File: ui/src/app/components/confirm-workspace-delete-modal.tsx [hunk 1]
```diff
@@ -71,26 +50,14 @@ export const ConfirmWorkspaceDeleteModal = ({
             access to the workspace. If you still wish to delete this workspace
             and all items within it, type DELETE below to confirm.
           </div>
-          {displayUserInput && (
+          {
```
Comment (jmthibault79) [inline ui/src/app/components/confirm-workspace-delete-modal.tsx:53]: nit: brackets are no longer needed

**Pull request merged**